src/components/Shared/Form/index.tsx

```diff
@@ -18,55 +18,53 @@ interface IConnectedActions {
 }
 
 interface FormProps {
-	saveHandler: () => void
+	dismissModal: () => void
 }
+
 type Props = IConnectedActions & IConnectedState & FormProps
 
 class FormComponent extends React.Component<Props, any> {
 	constructor(props: Props) {
 		super(props)
-		this.handleOnChange = this.handleOnChange.bind(this)
-		this.createNewOrg = this.createNewOrg.bind(this)
-		this.state = { name: '' }
+		this.state = this.initialState()
 	}
 
-	public initialState() {
+	public initialState = () => {
 		return { name: '' }
 	}
 
-	public handleOnChange(event: any): void {
+	public handleOnChange = (event: any) => {
 		this.setState({ name: event.target.value })
 	}
 
-	public createNewOrg(e: any) {
+	public createNewOrg = (e: any) => {
```

Comment:
https://stackoverflow.com/questions/37596405/how-to-set-the-type-of-a-window-event

Let's try to get a type on the event `e`